cdap-data-fabric/src/main/java/co/cask/cdap/store/StoreDefinition.java

```diff
@@ -661,4 +665,72 @@ public static void createTable(StructuredTableAdmin tableAdmin) throws IOExcepti
       tableAdmin.create(PROGRAM_LINEAGE_SPEC);
     }
   }
+
+  /**
+   *
+   */
+  public static final class FieldLineageStore {
```

Comment:
missing javadoc?

Reply:
Please talk about structure of the tables in the javadoc.

Reply:
done. I think the structure is straightforwards from the schema - endpoint_checksum is the only tricky one